Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Mathieu Besançon"]
 version = "0.1.0"
 
 [deps]
+BlockDiagonals = "0a1fb500-61f7-11e9-3c65-f5ef3456f9f0"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 
```

Comment:
that PR would be a good opportunity to add compat bounds on MOI and BlockDiagonals
MathOptInterface = "0.9"
BlockDiagonals = "0.1"

Reply:
(otherwise I can do it in a next PR)